inspire/modules/workflows/tasks/matching.py

```diff
@@ -248,4 +248,5 @@ def _update_old_object(obj, eng):
 def arxiv_set_category_field(obj, eng):
     """Temporary measure to enable sorting by primary category."""
     record = get_record_from_obj(obj, eng)
-    obj.uri = record.get("report_number.arxiv_category", [""])[0]
+    obj.uri = record['arxiv_eprints'][0]['categories'][0]
```

Comment:
I tried this and it works, so we can keep this version in the end:
record.get("arxiv_eprints.categories".
[""])[0]